src/libs/focusWithDelay/index.native.js

```diff
@@ -0,0 +1,6 @@
+import focusWithDelay from './focusWithDelay';
+
+/**
+ * We enable the delay on native to display the keyboard correctly
+ */
+export default focusWithDelay(false);
```

Comment:
No need to delay focus in android as it already works fine based on my testing.
If you also experience the same, I think this file should be renamed to `index.ios.js`.